### ISSUE TITLE
Installing issue with sinatra-2.1.0

### ISSUE THREAD

**Issue by yuta1024** [IzumiSy/sinatra-validation#13]
I was installing `sinatra-2.1.0` and `sinatra-validation`. However the version of `sinatra-validation` is old, the following errors occured in rspec.
> undefined method `Schema' for Dry::Validation:Module

I think the cause of this issue is below (`v0.1.0` does not set version constraints).
https://github.com/IzumiSy/sinatra-validation/blob/v0.3.1/sinatra-validation.gemspec#L24
https://github.com/IzumiSy/sinatra-validation/blob/v0.1.0/sinatra-validation.gemspec#L24

I was installing gems using below `Gemfile` and the result is following.
```
source 'https://rubygems.org'

gem 'sinatra'
gem 'sinatra-validation'
```

```
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies...
Using bundler 2.1.4
(snip)
Fetching sinatra 2.1.0
Installing sinatra 2.1.0
Fetching sinatra-contrib 2.1.0
Installing sinatra-contrib 2.1.0
Fetching sinatra-validation 0.1.0
Installing sinatra-validation 0.1.0
Bundle complete! 2 Gemfile dependencies, 20 gems now installed.
Bundled gems are installed into `./vendor/bundle`
```

 The current workaround is to use `sinatra-2.0.x`. Could you please support sinatra-2.1.0?

**Comment by IzumiSy** [comment]
Thank you for filing issues. Let me check this out.

**Comment by IzumiSy** [comment]
Released as 0.3.2

**Comment by yuta1024** [comment]
Thank you for supporting sinatra-2.1.0!
